docs/releases/6.0.txt

```diff
@@ -265,6 +265,28 @@ of each library are the first to add or confirm compatibility with Python 3.12:
 * ``sqlparse`` 0.5.0
 * ``tblib`` 3.0.0
 
+Custom ORM expressions should return params in a ``tuple``
+----------------------------------------------------------
+
+Prior to Django 6.0, :doc:`custom lookups </ref/models/lookups>` and
+:ref:`expressions <writing-your-own-query-expressions>` implementing the ``as_sql``
+method (and their underlying parts ``process_lhs`` and ``process_rhs``) were
+permitted to return a sequence of params in either a list or a tuple. To
+address the interoperatibility problems that resulted, the second element of
+``as_sql``'s return should now be a ``tuple``::
+
+  def as_sql(self, compiler, connection) -> tuple[list, tuple]: ...
+
+If your custom expressions support multiple versions of Django, you should
```

Comment:
By pointing this out I'm kind of detracting from the point @sarahboyce made above that we didn't strictly need to adjust the example for doing `lhs_params + rhs_params`, but during the transition period I think it's helpful to point out that doing so is not the best idea. But I can also see how this is "documenting a bug". Thoughts?

Reply:
I think it's worth documenting and I like the way it's phrased here. Having a location to point at in the release notes when the inevitable reports about mixing tuples and lists come in will be quite handy.